src/EPR.Calculator.Frontend/Controllers/UploadFileController.cs

```diff
@@ -76,7 +76,7 @@ public IActionResult DownloadCsvTemplate()
             try
             {
                 var filePath = Path.Combine(Directory.GetCurrentDirectory(), StaticHelpers.Path);
-                return PhysicalFile(filePath, "text/csv", "SchemeParameterTemplate.v0.1.xlsx");
+                return PhysicalFile(filePath, "text/csv", StaticHelpers.Path);
```

Comment:
Similar to the constant` MediaType = "application/json"`
Please create a constant for `"text/csv"`